case/src/main/kotlin/com/ritense/case/web/rest/dto/CaseTabUpdateOrderDto.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.case.web.rest.dto
+
+import com.ritense.case.domain.CaseTabType
+
+data class CaseTabUpdateOrderDto(
```

Comment:
This is the same as `CaseTabUpdateDto`, right? Can't we use the same one?

Reply:
There is a small difference. This dto also includes a `key` field